styles.css

```diff
@@ -0,0 +1,421 @@
+  font-weight: 100;
+}
+
+.bold-400 {
+  font-weight: 400;
+}
+
+.red-bg {
+  background-color: var(--red);
+}
+
+.text-red {
+  color: var(--red);
+}
+
+.font-sm {
+  font-size: 11px;
+}
+
+.gray {
+  color: var(--gray);
+}
+
+.light-gray-bg {
+  background-color: #f4f5f6;
+}
+
+.black {
+  color: var(--black);
+}
+
+.white {
+  color: white;
+}
+
+.subscribe-btn {
+  background-color: #ffe400;
+  border-radius: 24px;
+  font-size: 13px;
+  line-height: 14px;
+  padding: 4px 7px;
+  max-width: 150px;
+}
+
+.weather-img img {
+  height: 20px;
+  width: 20px;
+}
+
+img[alt="user"] {
+  width: 2rem;
+  height: 2rem;
+}
+
+img[alt="menu"] {
+  height: 18px;
+}
+
+img[alt="logo"] {
+  width: 132px;
+  height: 20px;
+}
+
+.small-title {
+  font-size: 14px;
+  font-weight: 700;
+}
+
+.news-title {
+  font-family: var(--roboto);
+  font-size: 22px;
+  line-height: 24px;
+  font-weight: 700;
+}
+
+.news-title a {
+  color: var(--black);
+  text-decoration: none;
+}
+
+.h3 a {
+  color: var(--black);
+  text-decoration: none;
+}
+
+.news-summary {
+  font-size: 13px;
+  color: #333;
+  font-weight: 400;
+}
+
+.text-overlay,
+.text-overlay-turn {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  background-color: rgba(255, 255, 255, 0.9);
+  padding: 0.5rem 1rem;
+  color: var(--red);
+  font-family: var(--roboto);
+  font-weight: 700;
+}
+
+.row {
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.profile-img {
+  border-radius: 50%;
+  width: 63px;
+  height: 63px;
+}
+
+.cover-h3 a {
+  line-height: 1px;
+}
+
+.links a {
+  color: rgb(102, 102, 102);
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 100;
+}
+
+.h3 a:hover,
+.nav-2 a:hover,
+.nav-3 a:hover {
+  color: #f92859;
+}
+
+.h3 span {
+  font-size: 14px;
+}
+
+.debate-section-top {
+  border-top: 4px solid #f92859;
+  border-bottom: 3px double #d9d9d9;
+}
+
+.debate-section-bottom {
+  border-bottom: 4px solid #1c87fb;
+}
+
+.text-gray-light {
+  color: #555;
+}
+
+.vs-btn p {
+  position: absolute;
+  background-color: var(--black);
+  color: #ccc;
+  padding: 5px 8px;
+  border-radius: 50%;
+  font-size: 14px;
+  font-weight: 700;
+  top: -1.2rem;
+}
+
+.signup-container {
+  background-color: #f4f5f6;
+  border: 1px solid #e5e5e5;
+  width: 300px;
+  font-family: var(--roboto);
+  font-size: 19px;
+  font-weight: bold;
+  padding: 20px;
+}
+
+.signup-container input {
+  outline: none;
+  padding-left: 5px;
+  border: 1px solid #ccc;
+}
+
+.signup-container input::placeholder {
+  color: #666;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 100;
+  font-size: 15px;
+}
+
+.signgup-img {
+  height: 30px;
+  width: 70px;
+}
+
+.magazine-img {
+  height: 30px;
+  width: 40px;
+}
+
+.signup-container span:first-child,
+.in-magazine {
+  width: fit-content;
+  padding: 4px 0;
+}
+
+.signup-container span:last-child {
+  line-height: 1;
+}
+
+.cover-text {
+  padding: 5px 7px;
+  color: white;
+}
+
+.cover-link:hover {
+  color: var(--red);
+}
```

Comment:
same style definition of the selector on line 70. Group selectors, or create new classes.

Reply:
Implemented this